### DISCUSSION TITLE
404 Not Found - GET https://repo.huaweicloud.com/repository/npm/@types/node/-/node-14.18.48.tgz

### DISCUSSION THREAD

**Discussion by VladimirAstafievInDrive** [cypress-io/cypress#26972] [General]
Cant install cypress with this error 404 Not Found - GET https://repo.huaweicloud.com/repository/npm/@types/node/-/node-14.18.48.tgz

**Comment by MikeMcC399** [comment]
- See resolved issue https://github.com/cypress-io/cypress/issues/26981
